app/src/main/java/com/sopt/now/compose/data/UserRepository.kt

```diff
@@ -1,51 +1,56 @@
 package com.sopt.now.compose.data
 
-import android.content.Context
 import android.content.SharedPreferences
 
-class UserRepository(context: Context) {
-    private val sharedPreferences: SharedPreferences =
-        context.getSharedPreferences("SaveLogin", Context.MODE_PRIVATE)
-
+class UserRepository(private val preferences: SharedPreferences) {
     fun setUserLoggedIn(loggedIn: Boolean) {
-        sharedPreferences.edit().putBoolean("isLoggedIn", loggedIn).apply()
+        preferences.edit().putBoolean("isLoggedIn", loggedIn).commit()
```

Comment:
```suggestion
        preferences.edit(commit = true) {
            it.putBoolean("isLoggedIn", loggedIn)
        }
```

- preferences-ktx 활용하시면 이렇게 적을 수도 있습니다.
- commit을 활용하신 이유가 있을까요?

Reply:
apply()가 다음 코드가 실행될 때 까지 데이터 저장이 완료 되었는지 보장할 수 없다는 피드백을 받아 commit() 활용했습니다!

Reply:
👍🏻